algebra-utils/src/fft/polynomial/sparse.rs

```diff
@@ -128,8 +128,8 @@ impl<F: Field> Into<DensePolynomial<F>> for SparsePolynomial<F> {
 #[cfg(test)]
 mod tests {
     use crate::{get_best_evaluation_domain, DensePolynomial, SparsePolynomial};
-    use crate::fields::mnt6753::fr::Fr;
-    use crate::Field;
+    use algebra::fields::bls12_381::fr::Fr;
```

Comment:
Is this a test only for bls12_381? What about the other curves? BN382 and Tweedle?

Reply:
Actually tests and benches need a huge refactoring: we should write single, templatized, test and bench functions and just call them with the appropriate instantiation. Look at: https://github.com/HorizenOfficial/ginger-lib/issues/76